RealmSwift/Tests/ObjectCreationTests.swift

```diff
@@ -70,54 +70,6 @@ class ObjectCreationTests: TestCase {
             SwiftOptionalDefaultValuesObject.defaultValues(), boolObjectValue: true)
     }
 
-    func testInitWithDictionary() {
```

Comment:
Why are these tests deleted? They're still applicable and unrelated to this project.

Reply:
Skipping only invalid check for mixed property given that it now accepts lists and dictionaries